polls/views.py

```diff
@@ -0,0 +1,139 @@
+from django.views import View
+from django.http import HttpResponse
+
+import random
+import datetime
+
+from rest_framework import generics
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from polls.models import *
+from accounts.views import CustomUser
+
+# Create your views here.
+
+#create polls for testing
+def create_polls(user: CustomUser):
+    def get_organizations():
+        orgs = Company.objects.all()
+        count = orgs.count()
+        while count < 5:
+            org = Company(name="Firma nr " + str(count))
```

Comment:
You can use `Company.objects.create(...)` to simplify this

https://docs.djangoproject.com/en/3.1/ref/models/querysets/#create